docs/zed/README.md

```diff
@@ -1,98 +1,288 @@
+  + [1.5 Time Travel](#15-time-travel)
+  + [1.6 Search Indexes](#16-search-indexes)
+    - [1.6.1 Index Rules](#161-index-rules)
+    - [1.6.2 Indexing Workflows](#162-indexing-workflows)
+* [2. Zed Commands](#2-zed-commands)
+  + [2.1 Auth](#21-auth)
+  + [2.2 Branch](#22-branch)
+  + [2.3 Create](#23-create)
+  + [2.4 Delete](#24-delete)
+  + [2.5 Drop](#25-drop)
+  + [2.6 Index](#26-index)
+    - [2.6.1 Index Apply](#261-index-apply)
+    - [2.6.2 Index Create](#262-index-create)
+    - [2.6.3 Index Drop](#263-index-drop)
+    - [2.6.4 Index Ls](#264-index-ls)
+    - [2.6.5 Index Update](#265-index-update)
+  + [2.7 Init](#27-init)
+  + [2.8 Load](#28-load)
+  + [2.9 Log](#29-log)
+  + [2.10 Merge](#210-merge)
+  + [2.11 Query](#211-query)
+  + [2.12 Rename](#212-rename)
+  + [2.13 Serve](#213-serve)
+  + [2.14 Use](#214-use)
+
+## 1. The Lake Model
+
+A Zed lake is a cloud-native arrangement of data, optimized for search,
+analytics, ETL, data discovery, and data preparation
+at scale based on data represented in accordance
 with the [Zed data model](../formats).
 
-### Lake Environment
+A lake is organized into a collection of data pools forming a single
+administrative domain.  The current implementation supports
+ACID append and delete semantics at the commit level while
+we have plans to support CRUD updates at the primary-key level
+in the near future.
 
-The ZED_LAKE environment variable indicates the location of
-the Zed lake data store managed by the `zed` command.
-If ZED_LAKE is a file or S3 URI, then the `zed` command interacts
-directly with the storage endpoint specified by that URI.
-If ZED_LAKE is an http URL, then `zed` acts as a client and sends
-all of its actions to a service that implements the Zed lake API.
-
-The `-lake` flag overrides the ZED_LAKE environment and defaults
-to `http://localhost:9867` if neither ZED_LAKE exists or `-lake`
-is provided.
+The semantics of a Zed lake loosely follows the nomenclature and
+design patterns of `git`.  In this approach,
+* a _lake_ is like a GitHub organization,
+* a _pool_ is like a `git` repository,
+* a _branch_ of a _pool_ is like a `git` branch,
+* the _use_  command is like a `git checkout`, and
+* the _load_ command is like a `git add/commit/push`.
 
-A Zed lake is made available as a service with the `zed serve` command.
-In this case, the service presents the configured lake (`-lake` or ZED_LAKE)
-via the Zed lake API on the indicated port (default 9867).
+A core theme of the Zed lake design is _ergonomics_.  Given the Git metaphor,
+our goal here is that the Zed lake tooling be as easy and familiar as Git is
+to a technical user.
 
-Concurrent access to any Zed storage endpoint, of course, preserves
+Since Zed lakes are built around the [Zed data model](../formats/zed.md),
+getting different kinds of data to into and out of a lake is easy.
+There is no need to define schemas or tables and fit
+semi-structured data into schemas before loading data into a lake.
```

Comment:
```suggestion
There is no need to define schemas or tables or fit
semi-structured data into schemas before loading data into a lake.
```

Reply:
That's not quite right... I changed to "and then fit".  Still a little awkward but I'm getting tired :-)